admin-dev/themes/new-theme/js/pages/catalog-price-rule/form/index.ts

```diff
@@ -24,19 +24,40 @@
  */
 
 import IncludeTaxFieldToggle from '@components/form/include-tax-field-toggle';
+import CurrencySymbolUpdater from '@components/form/currency-symbol-updater';
 import PriceFieldAvailabilityHandler from './price-field-availability-handler';
 
 import CatalogPriceRuleFormMap from './catalog-price-rule-form-map';
 
 const {$} = window;
 
 $(() => {
+  new CurrencySymbolUpdater(
```

Comment:
Why not include this component directly inside the `IncludeTaxFieldToggle` instead of duplicating the initialization for both use cases? This would make `IncludeTaxFieldToggle` much easier to use

Reply:
Because it's not the scope of the `ReductionTaxFieldToggle` to update the currency symbol.